inline-r/src/Language/R/QQ.hs

```diff
@@ -67,7 +58,7 @@ r = QuasiQuoter
 -- | Construct an R expression but don't evaluate it.
 rexp :: QuasiQuoter
 rexp = QuasiQuoter
```

Comment:
As pointed out by @qnikst , `rexp` is now completely useless so maybe we should just remove it all-together?